src/components/HirerProfileLayout.tsx

```diff
@@ -14,7 +14,7 @@ function HirerProfileLayout({ children, className, step }: ContainerProps) {
 
   return (
     <div className='bg-white text-black'>
-      <header className='navbar w-full border-stroke bg-white duration-300'>
+      {step > 0 && <header className='navbar w-full border-stroke bg-white duration-300'>
```

Comment:
this component his made for onboarding, not clean to use it in the admin and add this kind of edge case

Reply:
Created an admin settings layout component, it has an additional "go back" feature